core/lib/GlobalPool.ts

```diff
@@ -90,17 +82,12 @@ export default class GlobalPool {
     for (const { promise } of this.waitingForAvailability) {
       promise.reject(new Error('Shutting down'));
     }
-    if (this.chromeCore) {
-      await this.chromeCore.close();
-    }
-    if (this.mitmServer) {
-      await this.mitmServer.close();
-    }
-    SessionsDb.shutdown();
-    this.waitingForAvailability = [];
+    this.waitingForAvailability.length = 0;
```

Comment:
Why this change to `length = 0`?

Reply:
It's probably actually unnecessary. I was just keeping the same instance instead of potentially creating a new one in between "awaits" for closing.